backend/seeds/seeds.go

```diff
@@ -305,9 +305,9 @@ func setupAdminUser(store store.Store, username, password string) error {
 	}
 
 	admin := &types.User{
-		Username:     username,
-		PasswordHash: hash,
-		Groups:       []string{"cluster-admins"},
+		Username: username,
+		Password: hash,
+		Groups:   []string{"cluster-admins"},
```

Comment:
This will work, but I wonder if we should also set `PasswordHash` here.

Reply:
Added!